app/deepseg.cc

```diff
@@ -328,64 +328,64 @@ int main(int argc, char* argv[]) try {
 	printf("(c) 2021 by floe@butterbrot.org & contributors\n");
 	printf("https://github.com/floe/backscrub\n");
 	timinginfo_t ti;
-	ti.bootns = timestamp();
-	int debug  = 0;
-	bool showProgress = false;
-	bool showBackground = true;
-	bool showMask = true;
-	bool showFPS = true;
-	bool showHelp = false;
-	size_t threads= 2;
-	size_t width  = 640;
-	size_t height = 480;
-	const char *back = nullptr;
-	const char *vcam = "/dev/video1";
-	const char *ccam = "/dev/video0";
-	bool flipHorizontal = false;
-	bool flipVertical   = false;
-	int fourcc = 0;
+	ti.bootns            = timestamp();
+	int debug            = 0;
+	bool showProgress    = false;
+	bool showBackground  = true;
+	bool showMask        = true;
+	bool showFPS         = true;
+	bool showHelp        = false;
+	size_t threads       = 2;
+	size_t width         = 640;
+	size_t height        = 480;
+	const char *back     = nullptr;
+	const char *vcam     = "/dev/video1";
+	const char *ccam     = "/dev/video0";
+	bool flipHorizontal  = false;
+	bool flipVertical    = false;
+	int fourcc           = 0;
```

Comment:
Not quite a fan of those tabulated indentations.

Reply:
eww, those were supposed to be spaces..

Reply:
Yes. I'd prefer just `var = value;` with single spaces. Having the values all aligned at one column forces changes too all lines if any of those lines had its alignment changed.